src/geo/projection/globe.js

```diff
@@ -282,6 +243,56 @@ export default {
     createTileTransform(tr: Transform, worldSize: number): Object {
         return new GlobeTileTransform(tr, worldSize);
     },
+
+    farthestPixelDistance(tr: Transform): number {
+        const pixelsPerMeter = this.pixelsPerMeter(tr.center.lat, tr.worldSize);
+        // Find farthest distance of the globe that is potentially visible to the camera.
+        // First check if the view frustum is fully covered by the map by casting a ray
+        // from the top left/right corner and see if it intersects with the globe. In case
+        // of no intersection we need to find distance to the horizon point where the
+        // surface normal is parallel to the camera forward direction.
```

Comment:
"parallel" should be "perpendicular". Looks like this is wrong in the gl-native as well